README.md

```diff
@@ -1,44 +1,60 @@
-![cf](http://i.imgur.com/7v5ASc8.png) Lab 15- Binary Tree Data Structure
-====
-## To Submit this Assignment
-* Read this document entirely and estimate how long this assignment will take.
-* Work in a fork of this repository
-* Work in a branch on your fork
-* Protect your repository's `master` branch by activating `continuous-integration/travis-ci` status checks
-* Create a pull request from your `lab` + `<your name>` branch to your `master` branch
-* Open a pull request to this repository
-* Submit on canvas a question and observation,your original estimate, how long you spent, and a link to your pull request
-
-## Requirements
-#### Configuration
-<!-- list of files, configurations, tools, etc that are required -->
-  Your lab directory must include
-  * **README.md** -- with a documentation about your lab
-  * **.gitignore** -- with a robust .gitignore
-  * **.eslintrc** -- with the class .eslintrc file
-  * **.eslintignore** -- with the class .eslintignore
-  * **.package.json** -- with all dependencies and dev-dependencies
-  * **lib/** -- directory for holding your programs helper modules
-  * **__test__/** -- directory for holding your programs unit and integration tests
-  
-#### Testing
-* write at least three test assertions for each method of the Binary Tree Data Structure
-* organize your tests into appropriate describe/test blocks for test output readability
-
-#### Feature Tasks  
-* Implement an **in-order** traversal function that returns a string of the visited nodes.
-  * For example : `6,8,7,9,2,1,4,3,5`
-
-####  Documentation
-  * in your README, write documentation for you data structures
-  * your documentation should includes code block useage examples
-  * provide instructions for:
-    * installing and using your data structure
-    * accessing each method
-    * running your tests
-
-#### Rubric:
-  * Tests: 2pts
-  * Passes linter: 1pts
-  * Completed Data Structure: 5pts
-  * Big-O notation: 2pt
+# - 15 - Binary Tree
+
+## Overview: 
+Implementation of an in-order traversal function for a binary tree data structure. Includes a `node` module to establish left and right values, a `binary-tree` module establishing the root property, and a `traversals` module detailing the different traversal path functions.
+
+### Node Module
+```
+  export default class Node {
+    constructor(value, left = null, right = null) {
+      this.value = value;
+      this.left = left;
+      this.right = right;
+    }
+  }
+```
+
+### Binary-Tree Module
+```
+  export default class BinaryTree {
+    constructor(root = null) {
+      this.root = root;
+    }
+  }
+```
+
+### Traversals Module
+```
+  let inOrderOutput = '';
+    const inOrderTraversal = (rootNode) => {
+      if (!rootNode) {
+        return undefined;
+      }
+      inOrderTraversal(rootNode.left);
+      inOrderOutput += rootNode.value;
+      inOrderTraversal(rootNode.right);
+      return inOrderOutput;
+    };
+```
+
+###Documentation:
```

Comment:
Good README, but read instructions carefully, forgot instructions on how to run tests per the instructions
https://github.com/codefellows-seattle-javascript-401d23/15-BinaryTree